app/components/provider_interface/status_box_components/conditions_not_met_component.rb

```diff
@@ -0,0 +1,33 @@
+module ProviderInterface
+  module StatusBoxComponents
+    class ConditionsNotMetComponent < ActionView::Component::Base
+      include ViewHelper
+      attr_reader :application_choice
+
+      def initialize(application_choice:)
+        @application_choice = application_choice
+      end
+
+      def candidate_name
+        application_choice.application_form.full_name
+      end
+
+      def rows
+        [
+          {
+            key: 'Status',
+            value: render(ProviderInterface::ApplicationStatusTagComponent.new(application_choice: application_choice)),
+          },
+          {
+            key: 'Course',
```

Comment:
We are looking to change this shortly so that provider name and course name are separate rows as they can be changed individually. This will more closely align with the work we're doing when the provider responds to the application initially - we're going to show radio buttons for make an offer, make an offer but change course, make an offer but change provider, make an offer but change location, reject application.

Reply:
@adamsilver I agree; I think they can still be separate rows even if we wrap them in a component though